src/index.ts

```diff
@@ -3,3 +3,4 @@ export * from "./columns/index.js";
 export * from "./item-pagination/index.js";
 export * from "./pixel-padding/index.js";
 export * from "./devvit-watermark/DevvitWatermark.js";
+export * from "./dev-mock-service/index.js";
```

Comment:
I didn't understand why only fetch was exposed. what about all your other mocks?

:no_bell: in my experiments, this should tree-shake in devvit apps that don't use the mocks fine.

Reply:
>I didn't understand why only fetch was exposed

Where?